packages/website/ts/components/onboarding/portal_onboarding_flow.tsx

```diff
@@ -77,6 +77,8 @@ class PlainPortalOnboardingFlow extends React.Component<PortalOnboardingFlowProp
                 updateOnboardingStep={this._updateOnboardingStep.bind(this)}
                 disableOverlay={this.props.screenWidth === ScreenWidths.Sm}
                 isMobile={this.props.screenWidth === ScreenWidths.Sm}
+                // This is necessary to ensure onboarding stays open once the user unlocks metamask and clicks away
+                disableCloseOnClickOutside={utils.getBrowserType() === BrowserType.Firefox}
```

Comment:
Can we try opera and brave as well?

Reply:
Added Opera (had same issue). Brave also seemed to have this issue but it's not as easy to detect so doesn't seem worth it.